modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,286 @@
+
+| https://issues.couchbase.com/browse/MB-42498[MB-42498^]
+| *Summary*: Fixed an issue where a function was stuck in the deploying state when a bucket was deleted and rebalance was triggered.
+
+| https://issues.couchbase.com/browse/MB-42497[MB-42497^]
+| *Summary*: Fixed a race condition between undeploy and vBucket restream on rollback which resulted in a panic.
+
+| https://issues.couchbase.com/browse/MB-38403[MB-38403^]
+| *Summary*: Fixed an issue to close N1QL iterators after garbage collection.
+|===
+
+==== Index Service and Views
+
+[#table_fixedissues_v662-gsi-views,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-45541[MB-45541^]
+| *Summary*: A stale vBucket map in projector caused stale=false scans to timeout during KV rebalance. This has been fixed.
+
+| https://issues.couchbase.com/browse/MB-44409[MB-44409^]
+| *Summary*: Fixed an issue where the DDLServiceMgr took a long time to build a partitioned index with replica.
+
+| https://issues.couchbase.com/browse/MB-43959[MB-43959^]
+| *Summary*: The cluster info cache refresh has been optimized by querying the buckets.uri endpoint only on a change in version hash.
+
+| https://issues.couchbase.com/browse/MB-43766[MB-43766^]
+| *Summary*: The index build tokens were cleaned up only during rebalance. This has been updated so the index build tokens are cleaned up periodically on index deletion and by the lifecycle manager's janitor as well.
+
+| https://issues.couchbase.com/browse/MB-43764[MB-43764^]
+| *Summary*: Fixed an issue where the partition index order was not honored when index projection included all keys and the document key.
+
+| https://issues.couchbase.com/browse/MB-43280[MB-43280^]
+| *Summary*: Fixed an issue where `listReplicaCount` took more than 10s and timed out.
+
+| https://issues.couchbase.com/browse/MB-43072[MB-43072^]
+| Fixed a delay in the connectBucket function that caused the indexer to projector connection time out. 
+|===
+
+==== Query Service
+
+[#table_fixedissues_v662-query,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-45273[MB-45273^]
+| *Summary*: Fixed an issue where the primary index scan was incorrectly used with query containing an OR condition.
+
+| https://issues.couchbase.com/browse/MB-44979[MB-44979^]
+| *Summary*: Connections that timeout on read are now discarded.
+
+| https://issues.couchbase.com/browse/MB-44331[MB-44331^]
+| *Summary*: Fixed an issue where ANSI JOIN with intersect scan did not return any results. 
+
+| https://issues.couchbase.com/browse/MB-43488[MB-43488^]
+| *Summary*: Fixed an issue where an adaptive index with UNNEST alias returned wrong results.
+
+| https://issues.couchbase.com/browse/MB-43384[MB-43384^]
+| *Summary*: Fixed an issue where setupSSL() failed and impacted query execution.
+
+| https://issues.couchbase.com/browse/MB-43301[MB-43301^]
+| *Summary*: Fixed an issue where bulk DELETE with USE KEYS did not delete all keys.
```

Comment:
@kaminijagtiani Does this text represent the up to date summary, or should this be updated?

Reply:
Should be updated.
Fixed an issue wherein making a copy of a document internally wasn't being done correctly. 
In case of update, delete, and merge using update or delete it could have resulted in a data loss by deleting a wrong document. In case of join, let, unnest it could have given wrong results- if we were joining on meta().id or projecting meta().id.

Reply:
Thanks Kamini.